### PR TITLE
Docs refactor

### PR DESCRIPTION
**Pull request by SteveOye**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+| `statementKey` | `String` | **required** |
+
+
+```dart
+CreditScoreResponse response; //returns an CreditScoreResponse object 
+
+try {
+  response =
+    await FlutterPericulum.generateCreditScore(
+    token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: I would prefer if the environment did not have the access token. We don't want customers to think putting the token as an environment variable is the right thing to do. We want them to obtain the access token from their own backend.

File: README.md [hunk 2]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+| `token` | `String` | **required** |
+| `statementKey` | `String` | **required** |
+
+```dart
+List<CreditScoreResponse> response; //returns an List<CreditScoreResponse> object 
+
+try {
+  response =
+    await FlutterPericulum.getExisitingCreditScore(
+    token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: same here.

File: README.md [hunk 3]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+| `statementKey` | `String` | **required** |
+
+
+```dart
+List<Transaction> response; //returns an List<Transaction> object 
+
+try {
+  response =
+    await FlutterPericulum.getStatementTransaction(
+    token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: same here.

File: README.md [hunk 4]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+```dart
+StatementResponse response; //returns an StatementResponse object 
 
-          response = 'Failed to get platform version.';
 
-        }
+try {
+  response =
+    await FlutterPericulum.getStatementAnalytics(
+    token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: same here.

File: README.md [hunk 5]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+| `token` | `String` | **required** |
+| `statementKey` | `String` | **required** |
+
+```dart
+List<AffordabilityResponse> response; //returns an List<AffordabilityResponse> object 
+
+try {
+  response =
+    await FlutterPericulum.getAffordability(
+    token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: same here.

File: README.md [hunk 6]
```diff
@@ -15,34 +18,161 @@ dependencies:
...
+```dart
+var response; //returns the statusCode
+onPressed: () async {
+  var data = IdentificationData(
+      identifierName: 'bvn', value: '8678');
+  var data2 = IdentificationData(
+      identifierName: 'nin', value: '8678');
+  response = await FlutterPericulum
+      .attachCustomerIdentificationInfromation(
+          token: "${dotenv.env['tokenKey']}",
```
Comment (zabeehpericulum) [inline README.md]: same here.

File: example/lib/main.dart [hunk 7]
```diff
@@ -93,18 +100,16 @@ class _MyAppState extends State<MyApp> {
...
-                                        debugPrint(value
-                                            .averageMonthlyLoanRepaymentAmount
-                                            .toString())
-                                      });
+                                  statementKey: 3,
+                                  dti: 2.0,
+                                  loanTenure: 30,
+                                  averageMonthlyTotalExpenses: 4000,
+                                  averageMonthlyLoanRepaymentAmount: 1000,
+                                  token: "${dotenv.env['tokenKey']}");
```
Comment (zabeehpericulum) [inline example/lib/main.dart:107]: same here, can you just make this an empty string. In the readme can you please link to the authentication flow diagram in the readme?

**Review by zabeehpericulum — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SteveOye) [reply README.md]: need more context

**Review by zabeehpericulum — APPROVED** [review]

**Pull request merged**